packages/clerk-js/src/ui/primitives/Button.tsx

```diff
@@ -126,6 +132,7 @@ type OwnProps = PrimitiveProps<'button'> & {
   loadingText?: string;
   isDisabled?: boolean;
   isActive?: boolean;
+  hoverAsFocus?: boolean;
```

Comment:
We should destructure this inside the button component, otherwise its going to end up in the actual dom element (and throw a react warning)